applications/CompressiblePotentialFlowApplication/custom_python/add_custom_processes_to_python.h

```diff
@@ -16,22 +16,22 @@
 
 
 // System includes
-
+#include <pybind11/pybind11.h>
 
 // External includes
 
 
 // Project includes
 #include "includes/define.h"
```

Comment:
you can remove te inclusion of the define, it is already included in define_python